Makefile

```diff
@@ -99,6 +117,14 @@ deploy: manifests kustomize ## Deploy controller to the K8s cluster specified in
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
 	$(KUSTOMIZE) build config/default | kubectl apply -f -
 
+.PHONY: dev-deploy
+dev-deploy:  ## Deploy controller dev image in the configured Kubernetes cluster in ~/.kube/config
+	mkdir -p config/dev && cp config/default/* config/dev
+	cd config/dev && kustomize edit set image open-component-model/ocm-controller=$(IMG):$(TAG) \
+	&& kustomize edit set image open-component-model/ocm-registry-server=ghcr.io/souleb/ocm/ocm-registry:latest
```

Comment:
`ghcr.io/souleb/ocm/ocm-registry:latest`
This should probably not reference us in any way. 🤔 
I know @phoban01 is in there somewhere too. :D We should switch to the official repos Balazs published.

Reply:
Maybe switch this to `localhost:5000` which is described in the readme using Kind anyways?

Reply:
Yes I'll remove this when the PR will be ready. It is easier to test this way for now.